src/main/kotlin/org/openmbee/mms5/routes/LockCreate.kt

```diff
@@ -293,14 +293,16 @@ fun Route.createLock() {
                     
                         graph mor-graph:Metadata {         
                             ?_refSource a ?__mms_refSourceClass ;
-                                mms:commit ?__mms_commitSource ;
-                                mms:snapshot ?baseSnapshot .
+                                mms:commit ?__mms_commitSource .
```

Comment:
can you double check this whole sparql update? it seems to me like this line (later in the file) should be the only one needed, but the whole thing looks like it will basically copy the existing commit's snapshot ?baseGraph to ?__mms_model graph